rasa/nlu/test.py

```diff
@@ -1270,14 +1270,12 @@ def get_eval_data(
         result = interpreter.parse(example.get(TEXT), only_output_properties=False)
 
         if should_eval_intents:
-            if rasa.nlu.classifiers.fallback_classifier.is_fallback_classifier_prediction(
-                result
-            ):
-                # Revert fallback prediction to not shadow the wrongly predicted intent
+            fallback_classifier = rasa.nlu.classifiers.fallback_classifier
```

Comment:
Can we rather do something like `from ... import ...`? Assigning the imports seems a bit intransparent to me